### ISSUE TITLE
Downloading Output Missing

### ISSUE THREAD

**Issue by sachnk** [git-lfs/git-lfs#2819]
Hey --

When I clone my repository that has Git LFS artifacts, I get different outputs depending on the environment.

On my MacBook, while cloning my repository, I get output like this:

```
Resolving deltas: 100% (2486/2486), done.
Downloading mylib.so (13 KB)
```

But then on my CentOS 7 box, I get the same output but without the "Downloading ..." output.

I've relied on the "Downloading ..." output as a cue to confirm what files are actually tracked by Git LFS. Without it, I erroneously assumed my CentOS box did not pull the artifacts correctly, when in fact it did, but just did not output any "Downloading ..." messages.

Any ideas?

My MacBook environment:

```
$ git lfs env
git-lfs/2.3.4 (GitHub; darwin amd64; go 1.9.1)
git version 2.13.6 (Apple Git-96)

LocalWorkingDir=
LocalGitDir=
LocalGitStorageDir=
LocalMediaDir=
LocalReferenceDir=
TempDir=/var/folders/mq/7_w7g42908g4s13c25sj71yh0000gn/T/git-lfs
ConcurrentTransfers=3
TusTransfers=false
BasicTransfersOnly=false
SkipDownloadErrors=false
FetchRecentAlways=false
FetchRecentRefsDays=7
FetchRecentCommitsDays=0
FetchRecentRefsIncludeRemotes=true
PruneOffsetDays=3
PruneVerifyRemoteAlways=false
PruneRemoteName=origin
LfsStorageDir=lfs
AccessDownload=none
AccessUpload=none
DownloadTransfers=basic
UploadTransfers=basic
git config filter.lfs.process = "git-lfs filter-process"
git config filter.lfs.smudge = "git-lfs smudge -- %f"
git config filter.lfs.clean = "git-lfs clean -- %f"
```

My CentOS 7 environment:

```
$ git lfs env
git-lfs/2.3.4 (GitHub; linux amd64; go 1.8.3; git d2f6752)
git version 2.15.1

LocalWorkingDir=
LocalGitDir=
LocalGitStorageDir=
LocalMediaDir=
LocalReferenceDir=
TempDir=/tmp/git-lfs
ConcurrentTransfers=3
TusTransfers=false
BasicTransfersOnly=false
SkipDownloadErrors=false
FetchRecentAlways=false
FetchRecentRefsDays=7
FetchRecentCommitsDays=0
FetchRecentRefsIncludeRemotes=true
PruneOffsetDays=3
PruneVerifyRemoteAlways=false
PruneRemoteName=origin
LfsStorageDir=lfs
AccessDownload=none
AccessUpload=none
DownloadTransfers=basic
UploadTransfers=basic
git config filter.lfs.process = "git-lfs filter-process"
git config filter.lfs.smudge = "git-lfs smudge -- %f"
git config filter.lfs.clean = "git-lfs clean -- %f"
```

**Comment by sachnk** [comment]
Downgrading from git 2.15.1 to 2.13.4 resolved the issue on my CentOS box. Incidentally, the Git LFS server implementation is by Atlassian's Bitbucket Server v5.7.0 (which is the latest as of this writing).

**Comment by technoweenie** [comment]
This is due to a small bug in the new filter process functionality in Git. Here's a [Git mailing list thread](https://public-inbox.org/git/20171204203647.30546-1-lars.schneider@autodesk.com/) where @larsxschneider proposes a fix. It looks like it'll make it to a future Git release.

If your project has a lot of LFS objects, you may be better off sticking to Git 2.15, as the filter process is much more efficient than the old clean and smudge filters in Git. 

If you want an alternate way to see what LFS objects have been downloaded, you can use `ls-files`:

```sh
$ git lfs ls-files
9252a75c94 * bin/again.bin
0263829989 * bin/b.bin
98ea6e4f21 - bin/hi.bin
```

**Comment by lars-travis-test** [comment]
@sachnk The issue is fixed with Git 2.16 which was released today 🎉
